### PR TITLE
Re-Structure less file and classnames

### PR DESCRIPTION
**Pull request by SMahil**
This PR includes:

Restructure LESS files for better structure
Rename classnames according to component name
Add Prefix f8- to classnames
Update styles to match guidelines

### CODE REVIEW

**Comment by fabric8cd** [comment]
@SMahil fabric8/fabric8-ui:SNAPSHOT-PR-2137-1 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2137-fabric8-planner.badger.fabric8.io

**Comment by nimishamukherjee** [comment]
chrome

@SMahil the assignee image is 80x80. This needs to be resized.

I haven't blocked the PR as it is green.

**Comment by nimishamukherjee** [comment]
hover

@SMahil there are issues in the side panel
1. Hover has a white border at the bottom
2. There is no gap between the Iteration name and date Range

**Comment by nimishamukherjee** [comment]
title_cut

@SMahil the title and status box are cut in the quick preview
Overall padding for quick preview looks broken.

**Comment by nimishamukherjee** [comment]
horizontalscroll

@SMahil horizontal scroll on list view.

File: runtime/src/tests/work-item/work-item-list/page-objects/work-item-detail.page.js [hunk 1]
```diff
@@ -414,6 +414,8 @@ class WorkItemDetailPage {
     return element(by.id('comment_time_'+index)).getText();
   }
   commentEdit(index) {
+    browser.actions().mouseMove(element(by.css('#comment_body_' + index))).perform();
+    browser.wait(until.elementToBeClickable(element(by.css('#comment_body_' + index + ' .edit-icon'))), constants.WAIT, 'Failed to find comment edit button');
```
Comment (sanbornsen) [inline runtime/src/tests/work-item/work-item-list/page-objects/work-item-detail.page.js]: This line crosses the the maximum number of column standard, please break it in 2-3 lines.

File: runtime/src/tests/work-item/work-item-list/page-objects/work-item-list.page.js [hunk 2]
```diff
@@ -151,8 +151,8 @@ class WorkItemListPage {
     return parentElement.element(by.css('.workItemList_Delete'));
   }
   workItemKebabAssocateIterationButton (parentElement) {
-    browser.wait(until.presenceOf(parentElement.element(by.css('.workItemList_Iteration'))), constants.WAIT, 'Failed to find clickWorkItemKebabButton');
-    return parentElement.element(by.css('.workItemList_Iteration'));
+    browser.wait(until.presenceOf(parentElement.element(by.id('workItemList_Iteration'))), constants.WAIT, 'Failed to find clickWorkItemKebabButton');
```
Comment (sanbornsen) [inline runtime/src/tests/work-item/work-item-list/page-objects/work-item-list.page.js]: This line crosses the the maximum number of column standard, please break it in 2-3 lines.

Comment (sanbornsen) [reply runtime/src/tests/work-item/work-item-list/page-objects/work-item-list.page.js]: Why is this an ID ? This is an iterative component. I thought we decided on not to use ID on iterative component.

File: runtime/src/tests/work-item/work-item-list/page-objects/work-item-list.page.js [hunk 3]
```diff
@@ -285,12 +285,12 @@ class WorkItemListPage {
 
   /* Description element relative to a workitem */
   workItemDescription (workItemElement) {
-    return workItemElement.element(by.css(".workItemList_description")).getText();
+    return workItemElement.element(by.id("workItemList_description")).getText();
```
Comment (sanbornsen) [inline runtime/src/tests/work-item/work-item-list/page-objects/work-item-list.page.js]: Again, this is an iterative component, why the ID is being used?

File: runtime/src/tests/work-item/work-item-list/page-objects/work-item-list.page.js [hunk 4]
```diff
@@ -312,15 +312,15 @@ class WorkItemListPage {
...
   }
 
   workItemViewTitle (parentElement) {
-    return parentElement.element(By.css( ".list-group-item-heading.workItemList_title" ));
+    return parentElement.element(By.css( ".list-group-item-heading.f8-wi__list-title" ));
   }
 
   workItemViewDescription (parentElement) {
-    return parentElement.element(By.css( ".list-group-item-text.workItemList_description" ));
+    return parentElement.element(By.id( "workItemList_description" ));
```
Comment (sanbornsen) [inline runtime/src/tests/work-item/work-item-list/page-objects/work-item-list.page.js]: Again, this is an iterative component, why the ID is being used?

File: runtime/src/tests/work-item/work-item-list/page-objects/work-item-list.page.js [hunk 5]
```diff
@@ -586,17 +586,17 @@ class WorkItemListPage {
   }
   /* Access the Kebab 'move to top' element relative to its parent workitem */
   workItemKebabMoveToTopButton (parentElement) {
-    browser.wait(until.presenceOf(parentElement.element(by.css('.workItemList_MoveTop'))), constants.WAIT, 'Failed to find clickWorkItemKebabButton');
-    return parentElement.element(by.css('.workItemList_MoveTop'));
+    browser.wait(until.presenceOf(parentElement.element(by.id('workItemList_MoveTop'))), constants.WAIT, 'Failed to find clickWorkItemKebabButton');
```
Comment (sanbornsen) [inline runtime/src/tests/work-item/work-item-list/page-objects/work-item-list.page.js]: Again, this is an iterative component, why the ID is being used?

File: src/app/components/work-item-list-entry/work-item-list-entry.component.html [hunk 6]
```diff
@@ -4,29 +4,37 @@
...
               {{workItem.attributes['system.title']}}
             </p>
-            <a [routerLink]="[constructUrl(workItem)]"
-              class="fa fa-list-alt action-icon"></a>
+            <a class="fa fa-list-alt f8-action-icon"
+              [routerLink]="[constructUrl(workItem)]"></a>
         </div>
-        <div class="list-group-item-text workItemList_description dn">
+        <div class="list-group-item-text hide"
+          id="workItemList_description">
```
Comment (sanbornsen) [inline src/app/components/work-item-list-entry/work-item-list-entry.component.html]: Why is this an ID? This is an iterative component !!

File: src/app/components/work-item-list-entry/work-item-list-entry.component.html [hunk 7]
```diff
@@ -39,29 +47,66 @@
...
+				<li>
+          <a id="workItemList_MoveBottom"
+            (click)="onMoveToBottom($event)">
+              Move to Bottom
+          </a>
+        </li>
 				<li role="presentation" class="divider"></li>
-				<li><a class="workItemList_Iteration" (click)="associateIterationModal.open($event)">Associate with Iteration...</a></li>
+				<li>
+          <a id="workItemList_Iteration"
```
Comment (sanbornsen) [inline src/app/components/work-item-list-entry/work-item-list-entry.component.html]: Why is this an ID? This is an iterative component !!

**Comment by SMahil** [comment]
New PR for this #2138

**Pull request closed without merging**